src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      />
+
+      <h1 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
```

Comment:
This tag used for title of the full page, so you need to use the another tag here
```suggestion
      <h3 class="card__name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h3>
```